core/shared/src/main/scala/org/http4s/headers/Sec-WebSocket-Accept.scala

```diff
@@ -36,17 +37,16 @@ object `Sec-WebSocket-Accept` {
   def parse(s: String): ParseResult[`Sec-WebSocket-Accept`] =
     ParseResult.fromParser(parser, "Invalid Sec-WebSocket-Accept header")(s)
 
-  private[this] val tchar: Parser[Char] = charIn("=").orElse(digit).orElse(alpha)
-
-  private[this] val token: Parser[String] = tchar.rep.string
+  private[this] val token: Parser[String] =
+    (charIn("+/").orElse(digit).orElse(alpha).rep ~ char('=').rep0(0, 2)).string
```

Comment:
This identical parser is used for both `Sec-WebSocket-Accept` and `Sec-WebSocket-Key`, right?

Maybe we should move it here and share it?
https://github.com/http4s/http4s/tree/46f2c4780b405c316fa30fce7a476cb0c682d316/core/shared/src/main/scala/org/http4s/internal/parsing